cluegonauts/app/urls.py

```diff
@@ -20,4 +20,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path("clueless/", include("clueless.urls"), name="clueless"),
+    path("clueless/gameb/", include("clueless.urls"), name="gameb"),
```

Comment:
Could we rename this endpoint to just 'game'?

```suggestion
    path("clueless/game/", include("clueless.urls"), name="game"),
```